cmd/evm/runner.go

```diff
@@ -310,7 +314,7 @@ func runCmd(ctx *cli.Context) error {
 			logger.WriteTrace(os.Stderr, debugLogger.StructLogs())
 		}
 		fmt.Fprintln(os.Stderr, "#### LOGS ####")
-		logger.WriteLogs(os.Stderr, statedb.Logs())
+		logger.WriteLogs(os.Stderr, prestate.Logs())
```

Comment:
I think this should be `runtimeConfig.State`.  We want logs from the modified state.  not the prestate.